python/datashaper/datashaper/engine/verbs/verbs_mapping.py

```diff
@@ -31,6 +43,98 @@ def inner(func: Callable[..., TableContainer]) -> Callable[..., TableContainer]:
     return inner
 
 
+def new_row(old_tuple: tuple, new_column_name: str, value: Any) -> tuple:
+    """Helper function to return rows in row-wise operations."""
+    old_named_tuple_type = type(old_tuple)
+    return namedtuple(
+        old_named_tuple_type.__name__, old_named_tuple_type._fields + (new_column_name,)
+    )(*(old_tuple + (value,)))
+
+
+def parallel_verb(
+    name: str,
+    treats_input_tables_as_immutable: bool = False,
+    override_existing: bool = False,
+    asyncio_type: str = "asyncio",

```

Comment:
Can we make this a string enum? Let's start with "asyncio" and "asyncio_threads"